docs/targets/solana.rst

```diff
@@ -217,18 +217,42 @@ the provided seeds, along with a seed bump. See the Solana documentation on
         }
     }
 
-Using spl-token
-_______________
 
-`spl-token <https://spl.solana.com/token>`_ is the solana native way of creating tokens, minting, burning and
-transfering token. This is the Solana equivalent of
+
+Solana Library
+______________
+
+In Solang's Github repository, there is a folder called ``solana-library``. It contains libraries for Solidity contracts
+to interact with Solana specific instructions. Currently, there are two libraries there. One for SPL tokens and another
+for Solana's system instructions. In order to use those functionalities, one must copy the correspondent library
+file to one's project folder and import the library.
+
+SPL-token
++++++++++
+
+`spl-token <https://spl.solana.com/token>`_ is the Solana native way of creating tokens, minting, burning and
+transferring token. This is the Solana equivalent of
 `ERC-20 <https://ethereum.org/en/developers/docs/standards/tokens/erc-20/>`_ and
 `ERC-721 <https://ethereum.org/en/developers/docs/standards/tokens/erc-721/>`_. We have created a library ``SplToken`` to use
```

Comment:
`We have created` is unnecessary for documentation. We have created all solang, after all.